docs/intentReaderHandlers/RegisterProviderHandler.md

```diff
@@ -0,0 +1,126 @@
+# RegisterProviderHandler 
+
+## Overview
```

Comment:
Change the first paragraph to below

RegisterProviderHandler is invoked when the task specified in the intent has the value "registerProviderHandler". This handler helps to inform FCA to use test Providers instead of the actual provider files. Test provider can be used to manipulate the different values to be returned for api's like keyboard, profile which requires a user input. Test provider files can be found here.

Reply:
Full usage of this handler can be as explained below. 
Use case - Requirement is to return abc@gmail.com for keyboard.email api 

- Register for the Keyboard Test Provider by sending the below intent - src/providers/KeyboardUIProviderTest.js
- Set the response for keyboard.email using the SetApResponse Handler using the below intent
- Call Keyboard.email api from UI

Reply:
the first line should still be present in the first paragraph.
**_RegisterProviderHandler is invoked when the task specified in the intent has the value "registerProviderHandler"._**